setup.py

```diff
@@ -4,7 +4,7 @@
     long_description = fh.read()
 
 setup(
-    name="osidb-bindings",
+    name="osidb_bindings",
```

Comment:
I think this should be OK, PYPI does some normalization and when there is a package with underscores in name it just converts them to dashes, so I think we are safe on this.